src/domain/app/navigation/Navigation.tsx

```diff
@@ -17,27 +25,58 @@ function Navigation() {
   const navigate = useNavigate();
   const location = useLocation();
 
+  const cmsLanguageOptions = useCmsLanguageOptions();
+
+  const staticMenuItems = useStaticLinks();
+
+  const getHref = useCallback(
+    (language: LanguageCodeEnum) => {
+      const nav = cmsLanguageOptions?.find((cmsLanguageOption) => {
+        return (
+          cmsLanguageOption.locale?.toLowerCase() === language.toLowerCase()
+        );
+      });
+      const strippedPathname = stripLocaleFromUri(location.pathname);
+      // special logic for static urls
+      if (Boolean(strippedPathname)) {
+        const staticUrl = staticMenuItems.find((menuItem) => {
+          return menuItem.slug?.startsWith(strippedPathname);
+        });
+        if (staticUrl) {
+          return `/${language.toLowerCase()}${staticUrl.slug || ''}`;
```

Comment:
```suggestion
          return `/${language.toLowerCase()}${staticUrl.slug ?? ''}`;
```
Sonarcloud might give some code smell/warning/note about preferring `x ?? y` instead of `x || y` so that's why I'm suggesting the use of `??` here instead of `||`